frontend/src/components/form/Form.tsx

```diff
@@ -0,0 +1,159 @@
+  const { Title } = Typography;
+  return (
+    <div>
+      <Title>Adding New User</Title>
+      <StyledForm
+        form={form}
+        name="add-new-user"
+        layout="vertical"
+      >
+        <div className="first-and-last-name-form">
+          <StyledForm.Item
+            className="form-item"
+            label="First name"
+            name="firstname"
+            required
+            tooltip="Required field"
+            rules={[
+              {
+                required: true,
+                message: 'Please enter your first name!',
+                whitespace: false,
+                max: 256,
+              },
+            ]}
+          >
+            <Input placeholder="First Name" />
+          </StyledForm.Item>
+          <StyledForm.Item
+            className="form-item"
+            label="Last name"
+            name="lastname"
+            required
+            tooltip="Required field"
+            rules={[
+              {
+                required: true,
+                message: 'Please enter your last name!',
+                whitespace: false,
+                max: 256,
+              },
+            ]}
+          >
+            <Input placeholder="Last Name" />
+          </StyledForm.Item>
+        </div>
+        <StyledForm.Item
+          className="form-item"
+          label="Email"
+          name="email"
+          required
+          tooltip="Required field"
+          rules={[
+            {
+              required: true,
+              message: 'Please enter your email!',
+              type: 'email',
+              max: 256,
+            },
+          ]}
+        >
+          <Input placeholder="Email" />
+        </StyledForm.Item>
+        <StyledForm.Item
+          className="form-item"
+          label="Password"
+          name="password"
+          required
+          tooltip="Required field"
+          rules={[
+            {
+              required: true,
+              message: 'Please enter your password',
+              max: 256,
+            },
+          ]}
+        >
+          <Input.Password placeholder="Password" />
+        </StyledForm.Item>
+        <StyledForm.Item
+          className="form-item"
+          label="Confirm password"
+          name="confirmpassword"
+          required
+          tooltip="Required field"
+          rules={[
+            {
+              required: true,
+              message: 'Please confirm your password',
+              max: 256,
+            },
+            ({ getFieldValue }) => ({
+              validator(_, value) {
+                if (!value || getFieldValue('password') === value) {
+                  return Promise.resolve();
+                }
+
+                return Promise.reject('Passwords do not match!');
+              },
+            }),
+          ]}
+        >
+          <Input.Password placeholder="Confirm password" />
+        </StyledForm.Item>
+        <StyledForm.Item className="submit-btn-parent">
```

Comment:
`StyledForm.Item` powinien tylko zawierać w sobie pole, które możemy uzupełnić. Tutaj mamy przycisk, także cały ten komponent jest do usunięcia. Zostaw tylko `Row` i jego zawartość. Jeśli się nie styluje odpowiednio to znaczy, że coś inaczej trzeba zrobić :p